modules/codemod/index.js

```diff
@@ -4,9 +4,27 @@
 const {spawn} = require('child_process');
 const chalk = require('chalk');
 
-const {_: commands, path} = require('yargs')
+const {_: commands, path, ignoreConfig, ignorePattern, verbose } = require('yargs')
   .scriptName('canvas-kit-codemod')
   .usage(chalk.bold.blueBright('$0 <transform> [path]'))
+  .options({
+    'ignore-config': {
+      type: 'string',
+      describe: 'Ignore files if they match patterns sourced from a configuration file (e.g. a .gitignore)',
+      default: '',
+    },
+    'ignore-pattern': {
+      type: 'string',
+      describe: 'Ignore files that match a provided glob expression (e.g. **/dist/**)',
+      default: '**/node_modules/**',
+    },
+    verbose: {
+      type: 'number',
+      describe: 'Show more information about the transform process',
+      default: 0,
+      choices: [0, 1, 2]
+    },
+  })
```

Comment:
Putting these options here allows them to be available for all our codemod scripts.

Reply:
Options are pulls from the jscodeshift docs: https://github.com/facebook/jscodeshift#usage-cli